### PR TITLE
Fetch angular size.

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: modules/tests/jvm/src/main/resources/simbad-2MFGC6625.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (rpiaggio) [inline modules/tests/jvm/src/main/resources/simbad-2MFGC6625.xml:1]: This file was just re-fetched from Simbad.

File: modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala [hunk 2]
```diff
@@ -183,9 +186,35 @@ trait VoTableParser {
 
     val parseMagnitudes = adapter.parseMagnitudes(entries)
 
+    def parseAngularSize: ValidatedNec[CatalogProblem, Option[AngularSize]] = {
+
+      def parseDoubleMinutesOpt(field: FieldId): ValidatedNec[CatalogProblem, Option[Angle]] =
+        entries.get(field) match {
+          case Some(v) =>
+            parseDoubleValue(field.ucd, v)
+              .map(_ * 60 * 1e6)
```
Comment (cquiroz) [inline modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala:195]: Is it coming as a double in degrees? Maybe we can parse directly?

**Review by cquiroz — APPROVED** [review]
LGTM

File: modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala (hunk 2 above)
Comment (rpiaggio) [reply modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala:195]: It's coming in as double in minutes.

Comment (cquiroz) [reply modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala:195]: Why make it easy, right

Comment (rpiaggio) [reply modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala:195]: I'm adding a comment

**Pull request merged**